wiki/Tournaments/Zotcade/2019/en.md

```diff
@@ -255,8 +306,156 @@ The iDOL@NSTER is run by various osu! community members at UCI or visiting for t
+| **Ikeda Chizuru** ![][flag_US] | **3** | 2 | ![][flag_US] MyAngelNeptune | Played offline |
+
+##### Losers
+
+| Player 1 |  |  | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| Achao3 ![][flag_US] | 1 | **3** | ![][flag_US] **Slo** | [Multiplayer match](https://osu.ppy.sh/community/matches/51942573) |
+
+#### Seeding
+
+All of the players except ![][flag_US] Ikeda Chizuru played in a [seeding multiplayer match](https://osu.ppy.sh/community/matches/51939463). ![][flag_US] Ikeda Chizuru was having connection issues and played the pool offline instead (with supervision from a refereee).
+
+| Player | Aggregate placement |
+| :-- | :-- |
+| ![][flag_US] fieryrage | 12 |
+| ![][flag_US] death grips | 26 |
+| ![][flag_US] Mathyu | 30 |
+| ![][flag_US] Darin on osu | 34 |
+| ![][flag_US] Musket | 34 |
+| ![][flag_US] Ikeda Chizuru | 58 |
+| ![][flag_US] [KOR] Sora | 61 |
+| ![][flag_US] pishifat | 65 |
+| ![][flag_US] Sliceofpie | 72 |
+| ![][flag_US] YungCowboy | 77 |
+| ![][flag_US] MyAngelNeptune | 79 |
+| ![][flag_US] Slo | 80 |
+| ![][flag_US] Achao3 | 100 |
+
+### osu!mania
+
+#### Finals
+
+| Player 1 |  |  | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Crescendo PiaNo** ![][flag_US] | **6** | 2 | ![][flag_US] Trainer- | [Multiplayer match](https://osu.ppy.sh/community/matches/51944062) |
+
+#### Semifinals
+
+##### Winners
+
+| Player 1 |  |  | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Crescendo PiaNo** ![][flag_US] | **5** | 1 | ![][flag_US] Trainer- | [Multiplayer match](https://osu.ppy.sh/community/matches/51941547) |
+
+##### Losers
+
+| Player 1 |  |  | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Trainer-** ![][flag_US] | **5** | 0 | ![][flag_US] Ouchiee | [Multiplayer match](https://osu.ppy.sh/community/matches/51943569) |
+
+#### Round 2
+
+##### Winners
+
+| Player 1 |  |  | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Crescendo PiaNo** ![][flag_US] | **4** | 0 | ![][flag_US] Ouchiee | [Multiplayer match](https://osu.ppy.sh/community/matches/51940641) |
+| **Trainer-** ![][flag_US] | **4** | 3 | ![][flag_US] Ecal | [Multiplayer match](https://osu.ppy.sh/community/matches/51940114) |
+
+##### Losers
+
+| Player 1 |  |  | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Ecal** ![][flag_US] | **3** | 0 | ![][flag_US] Chest Flattener | [Multiplayer match](https://osu.ppy.sh/community/matches/51941682) |
+| **Ouchiee** ![][flag_US] | **5** | 0 | ![][flag_US] Ecal | [Multiplayer match](https://osu.ppy.sh/community/matches/51942291) |
+
+#### Round 1
+
+| Player 1 |  |  | Player 2 | Notes |
+| --: | :-: | :-: | :-- | :-- |
+| **Ouchiee** ![][flag_US] | **3** | 0 | ![][flag_US] Chest Flattener | [Multiplayer match](https://osu.ppy.sh/community/matches/51939935) |
+
 ## Ruleset
 
+<!-- The "procedure" section says that higher roll bans first / picks second, but this was flipped during the event
```

Comment:
nvm comment is wrong I'm just going to scrap image and fix any rules depending on it